src/controller/adminController.ts

```diff
@@ -0,0 +1,109 @@
+import { userEntity } from "../entity/user.entity";
+import {
+  ExceptionMessage,
+  HttpStatusCode,
+  HttpStatusMessage,
+  SuccessMessage,
+} from "../interface/enum";
+import { userService } from "../service/user.service";
+import { responseUitls } from "../utils/response.util";
+import { Request, Response } from "express";
+import { utils } from "../utils/utils";
+import { AcceptAny } from "../interface/type";
+import { adminE } from "../entity/admin.entity";
+import { CustomException } from "../utils/exception.utils";
+import { adminSessionE } from "../entity/admin.session.entity";
+import { nodeMailer } from "../provider/nodemailer/nodenmailer";
+import { redis } from "../provider/redis/redis";
+import { MAIL_SUBJECT } from "../constant/constants";
+import { error } from "console";
+class adminController {
+  home = async (req: Request, res: Response) => {
```

Comment:
what is this? signature methods under class? in class methods are best way of implementation something like this async home(req,res){}